packages/shared/lib/core/wallet/helpers/buildPersistedAssetFromMetadata.ts

```diff
@@ -0,0 +1,27 @@
+import { NotVerifiedStatus, TokenStandard } from '../enums'
+import { IErc20Metadata, IIrc30Metadata, IPersistedAsset } from '../interfaces'
+import { AssetVerification } from '../types'
+
+export function buildPersistedAssetFromMetadata(
+    tokenId: string,
+    metadata: IIrc30Metadata | IErc20Metadata,
```

Comment:
We should use the `TokenMetadata` type here as well

Reply:
Same comment as above 🙏🏼